src/main/java/edu/ie3/util/quantities/PowerSystemUnits.java

```diff
@@ -226,13 +231,17 @@ public class PowerSystemUnits extends Units {
   private static final HashSet<String> REGISTERED_LABELS = new HashSet<>();
 
   static {
-    // varh, kvarh, Mvarh are kept out of this because they register for the same units as Wh, kWh,
-    // MWh
```

Comment:
Just to document this and to avoid having to search for this again... The problem becomes apparent when debugging the new test.

![grafik](https://github.com/user-attachments/assets/f45792af-5287-4f9a-923b-6a8ae4a482e6)